### ISSUE TITLE
No specific icons in firefox

### ISSUE THREAD

**Issue by donald-roth** [brian-girko/bookmarks-commander#30]
Dear Brian,

in Chrome (Vivaldi) the bookmarks commander has all functions. Surprisingly, in Firefox 93 (on Linux 64bit, Debian11) no specific icons are displayed. This makes cleaning up and matching large bookmark lists very difficult. Will the icons also be available in the Firefox addon?

Regards
Ronald

![ff-bookmarks-commander_20211010](https://user-images.githubusercontent.com/92273818/136712614-563fd346-6458-449d-b305-b15052b26937.png)

**Comment by brian-girko** [comment]
True! in Firefox, there is no reliable way to get favicons! So the extension uses a generic icon. As of the next release, you can force the extension to use Google's remote service to get icons by setting `localStorage('remote', true);`. This is not recommended since it uses an online service and that is why there is no visual setting for it.
